code/modules/mob/living/simple_animal/abnormality/_tools/zayin/realizer.dm

```diff
@@ -1,87 +1,68 @@
 /obj/structure/toolabnormality/realization
-	name = "realization engine V1"
-	desc = "A strange artifact."
+	name = "realization engine"
+	desc = "An artifact used to find true potential within certains items."
 	icon_state = "realization"
-	var/output
-	var/realized = list()
-	var/stat_total
-	var/list/stats = list(FORTITUDE_ATTRIBUTE,
-			PRUDENCE_ATTRIBUTE,
-			TEMPERANCE_ATTRIBUTE,
-			JUSTICE_ATTRIBUTE)
+	var/static/realized_users = list()
+	/// Assoc list 'input = output'
+	var/list/output = list(
+		// ZAYIN
+		/obj/item/clothing/suit/armor/ego_gear/penitence = /obj/item/clothing/suit/armor/ego_gear/realization/confessional,
+		/obj/item/storage/book/bible = /obj/item/clothing/suit/armor/ego_gear/realization/prophet, // TEMPORARY
```

Comment:
We could just use the Paradise lost armor?
I mean it's fitting.

Reply:
Prophet, judging by the sprite, is plague doctor armor, not white night.